student-store-ui/src/components/ShoppingCart/ShoppingCart.jsx

```diff
@@ -1,19 +1,45 @@
 import * as React from "react"
 import "./ShoppingCart.css"; 
-//props: isOpen, products, shoppingCart
 
 export default function ShoppingCart({isOpen, shoppingCart}){
+    const isEmpty = Object.keys(shoppingCart).length === 0;
+    let subtotal = 0.00;
+
     return(
         <div className={isOpen ? "shopping-cart": "shopping-cart hidden"}>
-            <div className="header">
-                <h3>Shopping Cart</h3>
+            <h3>Shopping Cart</h3>
+            <p className={isEmpty ? "": "hidden"}>No itmes added to cart yet. Start shopping now!</p>
+            <div className={isEmpty ? "hidden": "table-sections"}>
+                <p>Name</p>
+                <p>Quantity</p>
+                <p>Unit Price</p>
+                <p>Cost</p>
+            </div>
+
+            {
+                Object.entries(shoppingCart).map( ([productName, productInfo]) => {
+                    const price = productInfo.price.toFixed(2);
+                    const quantity = productInfo.quantity;
+                    const total = (price * quantity).toFixed(2);
+                    subtotal += parseFloat(total);
+                    return(
+                        <div className="shopping-product" key={productName}>
+                            <p>{productName}</p>
+                            <p>{quantity}</p>
+                            <p>{price}</p>
+                            <p>${total}</p>
+                        
+                        </div>)
+                })
+            }
+            <div className={isEmpty ? "hidden": "receipt"}>
+                <p>Subtotal: ${subtotal.toFixed(2)}</p>
+                <p>Taxes and Fees: ${(subtotal * 0.09).toFixed(2)} </p>
```

Comment:
nit: Magic number 0.09. Maybe can defined a constant outside the component, with a descriptive name.